src/source/source_simple_resonator.ts

```diff
@@ -0,0 +1,12 @@
+import { source_component_base } from "src/source/base/source_component_base"
+import { z } from "zod"
+
+export const source_simple_resonator = source_component_base.extend({
+  ftype: z.literal("simple_resonator"),
+  frequency: z.union([z.string(), z.number()]),
```

Comment:
this should be "frequency", if it's not defined it should be a well-defined unit

Reply:
CC @imrishabh18 this is a tricky one to catch

Reply:
"1Hz" is a valid frequency for example, it's parsed into a standard unit `Hz`

Reply:
done.